### ISSUE TITLE
Support for model.json in Spark 

### ISSUE THREAD

**Issue by datasharkNL** [Azure/spark-cdm-connector#34]
In the documentation this is written about the model.json:

`Supports writing to CDM folders described by a manifest file only. Write support for model.json file is not planned.`

Please plan the support for model.json. For example the new provided functions for Dynamics 365 (Entity Stores to Data Lake and the coming feature Tables to Data Lake) generate model.json files instead of the other model files.

Power BI dataflow accepts only the model.json and not the newer version manifest. The newer version of Spark only generates "manifest" files and no more "model.json"

```
Yes, it does support model.json. For example:
.option("manifest", "/parquet/model.json")
```

Does it support writing as model.json? As far as I know it support model.json only as source. But for example Power BI dataflow supports only model.json and not a manifest file

(Opened an issue again as someone else posted their own issue in my previous closed issue. After which the issue was closed.)

**Comment by bissont** [comment]
Please comment @billgib

**Comment by billgib** [comment]
A Power BI connector has recently been released that enables Power BI to read CDM data described by manifest files.  You can find more information on how to use this here: https://powerbi.microsoft.com/en-us/blog/power-bi-desktop-april-2020-feature-summary/#_CDM_Folder_view

**Comment by SQLArchitect** [comment]
That's not enough.

They have to work with Dataflows.

**Comment by datasharkNL** [comment]
Indeed, the recently released Power BI connector is not suitable for Power BI Dataflows. Power BI dataflows accepts only model.json and not the new manifest files.

So the new version of CDM is now not usable with Power BI dataflows. 

And for example Dynamics new function does export a model.json. ([here](https://docs.microsoft.com/en-us/dynamics365/fin-ops-core/dev-itpro/data-entities/entity-store-data-lake).

If we want to combine this data with other sources, the output of this will be the new manifest file. (Mapping Dataflow of Azure Databricks with the new Spark (CDM) library.

**Comment by jinlinxu** [comment]
Up vote this one since PowerBI dataflows only support model.json
